lib/resources/r4/encounter.yaml

```diff
@@ -46,7 +44,6 @@ fields:
 
   - name: type
     required: 'Yes'
```

Comment:
double check type, system, value, class, the next three instances of type, and individual

Reply:
[Double checked](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-876b1ef3d3fb228190dea594ca0a12abR73) and added notes to clarify sub-fields are required when an optional field is given. [I also past tense'd a require](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-876b1ef3d3fb228190dea594ca0a12abR337)